app/src/main/resources/templates/casLoginView.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
```

Comment:
How does the screenreader understand the views? Is there any problem? We should consider accessibility as stated in the story.